Assets/scss/components/_list.scss

```diff
@@ -0,0 +1,108 @@
+.list{
+    background-color: var(--list-bg-color);
+    display: flex;
+    max-inline-size: 100%;
+    flex-direction: column;
+    font-family: $arial;
+    padding-inline: toRem(25);
+    padding-block: toRem(30);
+
+    @include min-screen(sm) {
+        padding-inline: toRem(60);
+        padding-block: toRem(55);
+    }
+    @include min-screen(md) {
+        padding-inline: toRem(90);
+        padding-block: toRem(70);
+    }
+    @include min-screen(lg) {
+        padding-inline: toRem(155);
+        padding-block: toRem(115);
+    }
+
+    &-head{
+        display: flex;
+        justify-content: space-between;
+        margin-block-end: toRem(16);
+        @include min-screen(md) {
+            margin-block-end: 0;
+        }
+    }
+
+    &-title{
+        color: var(--list-text-color);
+        font-size: toRem(35);
+
+        @include min-screen(md) {
+            font-size: toRem(30);
+        }
+    }
+    &-hideicon{
+        background: transparent;
+        border: none;
+        color: var(--list-showhide-icon-color);
+
+        @include min-screen(md) {
+            display: none;
+        }
+    }
+
+    .cmp-list{
+        padding-block: toRem(30);
+        border-block-start: toRem(1) solid var(--grey);
+        
+        @include min-screen(md) {
+            padding-block: toRem(20);
+            border: none;
+            display: grid;
+            grid-template-columns: repeat(2, 1fr);;
+            column-gap: toRem(20);
+
+            .cmp-list__item:nth-child(-n+2) {
+                border-block-start: toRem(1) solid var(--light-gray);
+                padding-block-start: toRem(30);
+            }
+        }
+
+        @include min-screen(lg) {
+            padding-block: toRem(30);
+            grid-template-columns: repeat(3, 1fr);;
+            
+            .cmp-list__item:nth-child(-n+3) {
+                border-block-start: toRem(1) solid var(--light-gray);
```

Comment:
@Satyam1RP  Border should be white in case of dark theme, with this method the color of border won't  change in case of dark theme
Cc:- @mathrik  @msharmaRP  @jvaishnavrp

Reply:
yes, it updated now.